src/PIL/Image.py

```diff
@@ -3607,18 +3607,36 @@ def _apply_env_variables(env=None):
 
 class Exif(MutableMapping):
     """
-    Exif class provides read and write access to EXIF image data.
+    This class provides read and write access to EXIF image data::
 
-    Only basic information is available on the root level, in Exif object
-    itself. In order to access the rest, obtain their respective IFDs using
-    :py:meth:`~PIL.Image.Exif.get_ifd` method and one of
-    :py:class:`~PIL.ExifTags.IFD` members (most notably ``Exif`` and
-    ``GPSInfo``).
+      from PIL import Image
+      im = Image.open("exif.png")
+      exif = im.getexif()  # Returns an instance of this class
+
+    Information can be read and written, iterated over or deleted::
+
+      print(exif[274])  # 1
+      exif[274] = 2
+      for k, v in exif.items():
+        print("Tag", k, "Value", v)  # Tag 274 Value 2
+      del exif[274]
```

Comment:
We probably want to show that the tag codes are integers, and I really like the conciseness of the example, but realistically, I think, no one would use random-looking numbers in real application code. Isn't it better to show it with an enum value?

```python
from PIL.ExifTags import Base
print(exif[Base.Orientation])  # 1
exif[Base.Orientation] = 2
for k, v in exif.items():
  print("Tag", Base(k).name, "Value", v)  # Tag Orientation Value 2
del exif[Base.Orientation]
```

Reply:
While the ExifTags enums are more helpful to humans, they are not as general - I expect that there are many tags that are not included in ExifTags, and I wouldn't want to limited users to just what we have listed.

I would hope that my last example in this docstring demonstrates the use of ExifTags sufficiently.